packages/common/rollup-configs.js

```diff
@@ -29,20 +31,21 @@ const getBanner = (pkg) => `/*
 /**
  * @returns {import('rollup').RollupOptions}
  */
-export const tsLibConfig = (pkg) => ({
-  input: 'src/index.ts',
+export const tsLibConfig = (pkg, inputFile) => ({
+  input: inputFile,
   output: {
     file: 'dist/index.js',
-    format: 'esm',
```

Comment:
I've realized that this is already the default Rollup output file format: https://rollupjs.org/guide/en/#outputformat